config.coffee

```diff
@@ -13,6 +13,8 @@ module.exports =
   API_URL: 'http://localhost:3000'
   POSITRON_URL: 'http://localhost:3005'
   FUSION_URL: null
+  REQUEST_LIMIT: 60
+  REQUEST_EXPIRE_MS: 60000 # 1 minute
```

Comment:
This probably is way too aggressive, but 👍  for making it a config var that we can tune up and down at any point.